frontend/src/components/explorer/gemBrowser/GemSearch.vue

```diff
@@ -144,12 +155,12 @@ export default {
   computed: {
     ...mapState({
       model: state => state.models.model,
-      resultsOrder: state => state.search.categories,
       searchTermString: state => state.search.searchTermString,
     }),
     ...mapGetters({
       models: 'models/models',
       searchResults: 'search/categorizedAndSortedResults',
+      componentTypeOrder: 'search/resultsComponentTypeOrder',
     }),
     placeholder() {
       return 'uracil, SULT1A3, ATP => cAMP + PPi, subsystem or compartment';
```

Comment:
Not sure if it is appropriate to raise this question in this PR. I think the examples given in the placeholder is not very good. 
1. the first three terms are detailed examples but the last one `subsystem or compartment` are types, better to be consistent
2. The example for the equation  `ATP => cAMP + PPi` might not be optimal, since the top ranked hits are not that relevant to this equation. I vaguely remember @MalinAhlberg has also raised this question previously.

Reply:
Nice that you noticed this! We think it makes sense to discuss this live at next standup. Another point to talk about is that if you start typing `ATP` the placeholder text disappear, i.e. you have to remember `ATP => cAMP + PPi` since you can't copy paste

Reply:
6943af784047ef93c66e4872ec5940e35d505b93 removes the placeholder and issue #897 covers adding a more appropriate one. Please resolve this comment if you think this covers the problems raised

Reply:
I see. make sense to me.